api/src/main/java/org/apache/cloudstack/api/command/user/iso/AttachIsoCmd.java

```diff
@@ -54,6 +54,9 @@
             required = true, description = "the ID of the virtual machine")
     protected Long virtualMachineId;
 
+    @Parameter(name=ApiConstants.FORCED, type=CommandType.BOOLEAN, description="If true, ejects existing iso before attaching on vmware. Default: false", since = "4.15.1")
```

Comment:
```suggestion
    @Parameter(name = ApiConstants.FORCED, type = CommandType.BOOLEAN, description ="If true, ejects existing ISO before attaching on VMware. Default: false", since = "4.15.1")
```